portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SearchAPI.testcase

```diff
@@ -81,19 +76,134 @@ definition {
 
 		task ("Run curl command to get keyword results") {
 			ACApi.getSearchKeywords(
-				displayLanguageId = "${displayLanguageId}",
 				faroBackendURL = "${faroBackendURL}",
-				groupId = "${groupId}",
 				osbAsahFaroBackendSS = "${osbAsahFaroBackendSS}",
 				osbAsahProjectId = "${osbAsahProjectId}",
-				page = "${page}",
-				size = "${size}",
-				sort = "${sort}");
+				parameters = "${parameters}");
 		}
 
 		task ("Assert response returns keyword") {
 			ACApi.assertSearchResponse(expected = "${keyword}");
 		}
 	}
 
+	@description = "Feature ID: LRAC-12058 | Automation ID: LRAC-12142 | Test Summary: Fetch keywords without optional parameters"
+	@priority = "3"
+	test RequestKeywordsUsingNoParameters {
+		property test.name.skip.portal.instance = "SearchAPI#RequestKeywordsUsingNoParameters";
+
+		var keyword1 = "without optional parameters 1";
+		var keyword2 = "without optional parameters 2";
+
+		var keywordList = "${keyword1},${keyword2}";
+
+		task ("Go to Server Administration") {
+			ServerAdministration.openServerAdmin();
+		}
+
+		task ("Get Faro Backend URL") {
+			var faroBackendURL = ACApi.getSearchHeaderValues(header = "liferayAnalyticsFaroBackendURL");
+		}
+
+		task ("Get OSB-Asah-Faro-Backend-Security-Signature") {
+			var osbAsahFaroBackendSS = ACApi.getSearchHeaderValues(header = "liferayAnalyticsFaroBackendSecuritySignature");
+		}
+
+		task ("Get OSB-Asah-Project-Id") {
+			var osbAsahProjectId = ACApi.getSearchHeaderValues(header = "liferayAnalyticsProjectId");
+		}
+
+		task ("Go to Site Page") {
+			ACUtils.navigateToSitePage(
+				pageName = "Home",
+				siteName = "guest");
+		}
+
+		task ("Type keyword in search bar") {
+			for (var keyword : list "${keywordList}") {
+				SearchPortlets.searchEmbedded(searchTerm = "${keyword}");
```

Comment:
Hey Vini, what do you think instead of creating a list with your keywords (line 98) you can just pass the keywords in the for loop and delete the list:
`for (var keyword : list "${keyword1},${keyword2}")`
what do you think?